cpp/log/cluster_state_controller.h

```diff
@@ -103,6 +104,7 @@ class ClusterStateController {
   void ClusterServingSTHUpdater();
 
   const std::shared_ptr<libevent::Base> base_;
+  const bool is_mirror_;                  // This log instance is a mirror
```

Comment:
Ah yes, this was originally used for some fiddling in `DetermineElectionParticipation` which I removed when I realised that whole method could bite the dust.
Dinged.